src/xmpp/message.c

```diff
@@ -98,6 +98,20 @@ static xmpp_stanza_t* _ox_openpgp_signcrypt(xmpp_ctx_t* ctx, const char* const t
 
 static GHashTable* pubsub_event_handlers;
 
+gchar*
+get_display_name(const ProfMessage* const message, gboolean* flags)
```

Comment:
I'm unsure why flags is a `gboolean` here. In the calling functions it's an `int` and:
```
// flags : 1st bit =  0/1 - me/not me. define: NO_                                                              
//         2nd bit =  0/1 - date/no date. define: NO_DATE
//         3rd bit =  0/1 - eol/no eol. define: NO_EOL
//         4th bit =  0/1 - color from/no color from. define: NO_COLOUR_FROM
//         5th bit =  0/1 - color date/no date. define: NO_COLOUR_DATE
//         6th bit =  0/1 - trusted/untrusted. define: UNTRUSTED
```

Reply:
Funny how the crippled glib types made this not even spit a warning ...